bert/src/lib.rs

```diff
@@ -57,8 +57,5 @@ pub use crate::{
     },
 };
 
-/// A Bert pipeline with an average pooler.
-pub type AvgBert = Pipeline<crate::tokenizer::bert::Tokenizer, AveragePooler>;
-
-/// A Roberta pipeline with an average pooler.
-pub type AvgRoberta = Pipeline<crate::tokenizer::roberta::Tokenizer, AveragePooler>;
+/// A Transformer pipeline with an average pooler.
+pub type AvgE5 = Pipeline<crate::tokenizer::huggingface::Tokenizer, AveragePooler>;
```

Comment:
❓ the pipeline is not really limited to e5, since we are able to load any transformer model. wdyt about `AvgTransformer` or `AvgEmbedder` or sth similar?